.github/dependabot.yml

```diff
@@ -25,6 +25,9 @@ updates:
       # Ignored intentionally since we have a GHA that updates to more
       # completely
       - dependency-name: "aws-sam-translator"
+      # Ignored intentionally since we have a GHA that updates to more
+      # completely
+      - dependency-name: "aws_lambda_builders"
```

Comment:
I removed it from the dependabot and add it to our GH action to run more frequently (every 4 hours instead of weekly)